src/lambda_function/function.py

```diff
@@ -1,58 +1,74 @@
 import boto3
-import json	
+import json
 import logging.config
 import os
 
-kms_key = os.environ['SSM_KMS_KEY_ARN']
+KMS_KEY = os.environ['SSM_KMS_KEY_ARN']
 client = boto3.client('ssm')
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 
 def handler(event, context):
 	logger.info('Event :{}'.format(event))
-	body = event['Body']
-	if 'Action' in body:
-		action = body['Action']
-		if action == 'GetParameter':
-			if 'Name' in body:
-				return get_parameter_from_store(body['Name'])
-			elif 'Names' in body:
-				return get_parameter_from_store(body['Names'])
-			else:
-				logger.info('Missing the parameter name in the event')
-		elif action == 'PutParameter':
-			return put_parameter_to_store(body['Name'], body['Type'], body['Value'])
-		else:
-			logger.info('Undefined action. Cannot complete the request')            					
+	if 'GetParameters' in event:
+		response = _get_parameter_from_store(event['GetParameters']['Names'])
+	elif 'GetParametersByPath' in event:
+		response = _get_parameter_from_store_by_path(event['GetParametersByPath']['Path'], event['GetParametersByPath']['Recursive'])
+	elif 'PutParameter' in event:
+		response = _put_parameter_to_store(event['PutParameter'])
 	else:
-		logger.info('Missing action (get/put) in event body')	
+		return 'Unexpected request. Cannot be completed.'
+	return response
+	
 
-		
-def get_parameter_from_store(name):
-	if isinstance(name, list):
-		parameter_list = []
-		for each_name in name:
-			params = {}
-			response = client.get_parameter(Name=each_name, WithDecryption=True)
-			params['Name'] = each_name
-			params['Value'] = response['Parameter']['Value']
-			parameter_list.append(params)
-		return parameter_list
+def _get_parameter_from_store(names):
+	parameters = {}
+	parameters_list = {}
+	for name in names:
+		params = client.get_parameter(Name=name, WithDecryption=True)
```

Comment:
Please change this to the `get_parameters` call. Why would we loop when it is provided for us?